examples/spring-boot-webflux-starter/src/test/java/example/springframework/boot/armeria/WebTestClientTest.java

```diff
@@ -0,0 +1,48 @@
+package example.springframework.boot.armeria;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.linecorp.armeria.spring.web.reactive.ArmeriaClientHttpConnector;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(
+        webEnvironment = WebEnvironment.RANDOM_PORT,
+        properties = "spring.main.web-application-type=reactive"
+)
+public class WebTestClientTest {
+
+    @RestController
+    static class HelloController {
```

Comment:
Shouldn't we test `example.springframework.boot.armeria.HelloController` rather than this one?

Reply:
Oops, my mistake. I copied the test code without thinking. Thanks!